redisearch/client_test.go

```diff
@@ -475,48 +489,58 @@ func TestClient_GetTagVals(t *testing.T) {
 	tags, err = c.GetTagVals("notexit", "tags")
 	assert.NotNil(t, err)
 	assert.Nil(t, tags)
+	teardown(c)
 }
 
 func TestClient_SynAdd(t *testing.T) {
 	c := createClient("testsynadd")
-
-	sc := NewSchema(DefaultOptions).
-		AddField(NewTextField("name")).
-		AddField(NewTextField("addr"))
-	c.Drop()
-	err := c.CreateIndex(sc)
-	assert.Nil(t, err)
-
-	gid, err := c.SynAdd("testsynadd", []string{"girl", "baby"})
-	assert.Nil(t, err)
-	assert.True(t, gid >= 0)
-	ret, err := c.SynUpdate("testsynadd", gid, []string{"girl", "baby"})
+	version, err := c.GetRediSearchVersion()
 	assert.Nil(t, err)
-	assert.Equal(t, "OK", ret)
+	if version <= 10699 {
```

Comment:
basically 1.8.99 ...

Reply:
Also its not like we do not support synonyms anymore, we just do not support synadd, we can still check synupdate.

Reply:
@MeirShpilraien , agree. Adding a new test called `TestClient_SynUpdate` so that we have `TestClient_SynAdd` (tested for redisearch <= 2.0, `TestClient_SynUpdate`, and `TestClient_SynDump`.